package.json

```diff
@@ -2,6 +2,9 @@
   "name": "ccd-api-gateway-web",
   "version": "1.4.0",
   "private": true,
+  "engines": {
+    "node": "^8.9.4"
```

Comment:
I believe if we don't put an upper limit to the version of node to say it has to be <10, we'll end up with environment using the latest v10 and Docker using v8; which would mean that the functional tests wouldn't be running on a prod-like environment. Thoughts?

Reply:
I don't think so:
```~/dev/moj/repos/ccd-admin-web (RDM-3073-align_nodejs_versions)$ node --version
v10.11.0
~/dev/moj/repos/ccd-admin-web (RDM-3073-align_nodejs_versions)$ yarn install
yarn install v1.9.4
[1/5] 🔍  Validating package.json...
error ccd-admin-web@1.0.0: The engine "node" is incompatible with this module. Expected version "^8.9.4".
error Found incompatible module```

```
yarn run v1.9.4
error Command "insatall" not found.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
~/dev/moj/repos/ccd-admin-web (RDM-3073-align_nodejs_versions)$ yarn install
yarn install v1.9.4
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
success Already up-to-date.
```

Reply:
True, true... end of the week
Though by the look of it I'm not the only one having a hard time:
> error Command "insatall" not found.